python/whylogs/experimental/core/metrics/udf_metric.py

```diff
@@ -179,6 +225,7 @@ def zero(cls, config: Optional[MetricConfig] = None) -> "UdfMetric":
 register_metric(UdfMetric)
 
 
+# _col_name_submetrics[schema_name][column_name] -> list (submetric_name, UDF)
```

Comment:
Remove commented out code, or explain in the comment why this is there?

Reply:
It's a hint for how to use the complicated data structure. I can add more context.